src/Fixer/PhpUnit/PhpUnitTestClassRequiresCoversFixer.php

```diff
@@ -80,6 +80,7 @@ protected function applyPhpUnitClassFix(Tokens $tokens, int $startIndex, int $en
             [
                 'phpunit\framework\attributes\coversclass',
                 'phpunit\framework\attributes\coversnothing',
+                'phpunit\framework\attributes\coversmethod',
```

Comment:
I believe we should also include `CoversFunction`.

Reply:
I believe so too, but should it be part of this PR?

Reply:
I think we can add it here too.